recipes/leftbar/group_row/group_row.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <dt-recipe-general-row
+    :description="names"
+    :unread-count="unreadCount"
+    :selected="selected"
+    v-on="$listeners"
```

Comment:
Not necessary, `v-on="$listeners"` is on the root element by default.

Reply:
Yeah it shouldn't be necessary. But by removing this, the click event isn't logged in the Actions in the SB panel. So I'm guessing it's not passing down the events to the root?

Reply:
Hmm wonder if it's different if the root is a vue component? it's fine just leave it then.